lib/routes/token.js

```diff
@@ -249,6 +251,20 @@ function confirmRefreshToken(params) {
       throw AppError.invalidScopes();
     }
     tokObj.scope = params.scope;
+
+    var now = new Date();
+    var lastUsedAt = tokObj.lastUsedAt;
+
+    if ((now - lastUsedAt) > REFRESH_LAST_USED_AT_UPDATE_AFTER_MS){
+      db.usedRefreshToken(encrypt.hash(params.refresh_token)).then(function() {
+        logger.debug('lastUsedAt updated to:', now);
```

Comment:
The logging system will turn the first argument into an `op`, and use the 2nd argument as the message details. Should be something like:

```js
logger.debug('refresh_token.lastUsedAt', now);
```